config/config.exs

```diff
@@ -32,12 +32,10 @@ config :plug, :mimes, %{
 }
 
 config :guardian, Guardian,
-  allowed_algos: ["HS512"], # optional
-  verify_module: Guardian.JWT,  # optional
   issuer: "CodeCorps",
   ttl: { 30, :days },
   verify_issuer: true, # optional
-  secret_key: "DELETE_ME",
+  secret_key: System.get_env("GUARDIAN_SECRET_KEY"),
```

Comment:
We may want to look more into this. The problem that we're going to run into is that when you build a release, this config gets compiled down into a `sys.config` file, which cannot read from `System.get_env/1`. This is true of both `distillery` and `exrm` releases. The recommended way to do runtime configuration in `distillery` is here:

https://github.com/bitwalker/distillery/blob/master/docs/Runtime%20Configuration.md

The problem with this is that we're not actually consuming this value ourselves, it's being consumed by Guardian. However, they do have an example using an anonymous function. So maybe we could do something like this:

```elixir

config :guardian, Guardian,
  secret_key: fn ->
    System.get_env("GUARDIAN_SECRET_KEY")
  end
```